### ISSUE TITLE
Version of docker required for `oc cluster up`

### ISSUE THREAD

**Issue by davidffrench** [aerogear/mobile-docs#220]
The `oc cluster up` prerequisite docs (https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#prerequisites) define that OpenShift 3.9 explicitly requires docker version 1.13. I am running docker CE version 17.09.0. Will this work, it is a little confusing to a newcomer and may be worth pointing out in our own docs to ignore this.

**Comment by pmacko1** [comment]
@davidffrench well... in the past I was told that the only working version is 17.09.1. Now `sudo dnf install docker` on fedora supposedly works

They are versioning the docker API as well. see https://docs.docker.com/develop/sdk/

I was convinced after some googling that the oclusterup docs are referring to deprecated feature which was versioned similarily too. Look for "Commercially supported Docker Engine"....
long story short, docker versioning is very confusing for the uninitiated.
17.09.0 is, i believe api 1.32

**Comment by davidffrench** [comment]
Thanks @pmacko1 . Ultimately this issue is not with our own docs but in the OpenShift prereq docs. In my opinion, they should be listed the CE version or at least saying, "Please run `docker version` and check the API version"

The only potential action on us from this issue would be if we want to call this out in our own docs almost as a workaround.

**Comment by pmacko1** [comment]
I have posted a pr on https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md to update the docs @davidffrench

**Comment by pmacko1** [comment]
i would leave this open for now, let's wait for openshift docs

**Comment by finp** [comment]
openshift issue, closing
